label_studio/users/api.py

```diff
@@ -66,7 +66,7 @@ def list(self, request, *args, **kwargs):
     @swagger_auto_schema(
         tags=['Users'],
         operation_summary='Create new user',
-        operation_description='Create a new user for Label Studio.',
+        operation_description='Create a user in Label Studio.',
         request_body=UserSerializer
```

Comment:
request_body is incorrect for this API endpoint, says requires org_membership, so something in the serializers needs to be updated, @makseq

Reply:
I was using the wrong API, ahhhh.